applications/crossbar/priv/couchdb/schemas/kapi.dialplan.bridge.json

```diff
@@ -0,0 +1,196 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "_id": "kapi.dialplan.bridge",
+    "description": "AMQP API for dialplan.bridge",
+    "properties": {
+        "Application-Name": {
+            "enum": [
+                "bridge"
+            ],
+            "type": "string"
+        },
+        "B-Leg-Events": {
+            "items": {
+                "enum": [
+                    "CALL_SECURE",
+                    "CALL_UPDATE",
+                    "CHANNEL_ANSWER",
+                    "CHANNEL_CREATE",
+                    "CHANNEL_DESTROY",
+                    "CHANNEL_EXECUTE",
+                    "CHANNEL_EXECUTE_COMPLETE",
+                    "CHANNEL_EXECUTE_ERROR",
+                    "CHANNEL_FAX_STATUS",
+                    "CHANNEL_INTERCEPTED",
+                    "CHANNEL_PROGRESS_MEDIA",
+                    "CHANNEL_REPLACED",
+                    "CHANNEL_TRANSFEREE",
+                    "CHANNEL_TRANSFEROR",
+                    "CHANNEL_BRIDGE",
+                    "CHANNEL_UNBRIDGE",
+                    "CHANNEL_HOLD",
+                    "CHANNEL_UNHOLD",
+                    "DETECTED_TONE",
+                    "DTMF",
+                    "LEG_CREATED",
+                    "LEG_DESTROYED",
+                    "RECORD_START",
+                    "RECORD_STOP",
+                    "dialplan"
+                ],
+                "type": "string"
+            },
+            "type": "array"
+        },
+        "Call-ID": {
+            "type": "string"
+        },
+        "Callee-ID-Name": {
+            "type": "string"
+        },
+        "Callee-ID-Number": {
+            "type": "string"
+        },
+        "Caller-ID-Name": {
+            "type": "string"
+        },
+        "Caller-ID-Number": {
+            "type": "string"
+        },
+        "Confirm-Cancel-Timeout": {
+            "type": "integer"
+        },
+        "Confirm-File": {
+            "type": "string"
+        },
+        "Confirm-Key": {
+            "type": "string"
+        },
+        "Continue-On-Fail": {
+            "type": "boolean"
+        },
+        "Custom-Channel-Vars": {
+            "type": "object"
+        },
+        "Custom-SIP-Headers": {
+            "type": "object"
+        },
+        "Dial-Endpoint-Method": {
+            "enum": [
+                "single",
+                "simultaneous"
+            ],
+            "type": "string"
+        },
+        "Enable-T38-Fax": {
+            "type": "string"
+        },
+        "Enable-T38-Fax-Request": {
+            "type": "string"
+        },
+        "Enable-T38-Gateway": {
+            "enum": [
+                "self",
+                "peer"
+            ],
+            "type": "string"
+        },
+        "Enable-T38-Passthrough": {
+            "type": "string"
+        },
+        "Endpoints": {
+            "items": {
+                "type": "string"
```

Comment:
yes, i was going to go through and update some to $ref'd objects. Probably do that now :)